packages/core-utils/src/fake-route-data.json

```diff
@@ -0,0 +1,5425 @@
+{
```

Comment:
Can we put these files in a test folder? that way they don't get bundled into a release

Reply:
I added a `.story` suffix to them which I think should prevent them from being bundled. An alternative idea is using the fake routes in the __mocks__ folder instead of creating this new set of fake data? Lmk what you think.

Reply:
the `.story` suffix is great! I'm agreed we should keep the story I think it's good. But the mock data needs to go in a dedicated testing folder! IS it possible to use the itinerary-body package as a guide?

Reply:
Added the fake data to a new mocks folder, that seemed to be how we handled it in itinerary body! Let me know if that's not what you were thinking